Tasks/VsTestV2/task.json

```diff
@@ -17,7 +17,7 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 2,
-        "Minor": 237,
+        "Minor": 238,
```

Comment:
will it be 238 or 2.237.1?

Reply:
It should be 238, because it's week 3 of a sprint